app/main.py

```diff
@@ -1,3 +1,70 @@
 class Distance:
-    # Write your code here
-    pass
+
+    def __init__(self, km: int) -> None:
+        self.km = km
+
+    def __str__(self) -> str:
+        return f"Distance: {self.km} kilometers."
+
+    def __repr__(self) -> str:
+        return f"Distance(km={self.km})"
+
+    def __add__(self, other: int | float) -> None:
```

Comment:
Still not fixed. `other` can be `Distance`. Fix everywhere.
```suggestion
    def __add__(self, other: int | float | Distance) -> None:
```